README.md

```diff
@@ -27,26 +27,18 @@ mvn package -DskipTests
 
 `mvn package` will assemble all the required dependencies and package into an uber jar.
 
-Pre-create Atlas models (Recommended)
-=====================================
+Create Atlas models
+===================
 
```

Comment:
Please add a clarifying comment on the top mentioning that the steps below are only needed for pre-Atlas 2.0. Post-Altas 2.0, these steps are **not** needed.

Reply:
I didn't add the specific version since there's no official version yet in Apache Atlas having this as built-in. I'm seeing several options here:

1. We may ask Apache Atlas community to roll out the new release, and specify the version once we have actual release. (either holding this PR or having follow-up PR)
2. We could just specify the supposed new version (2.1.0 I suppose). 

WDYT?

Reply:
Sure, let's specify the supposed next version for now. We can change it when it releases.